frontend/app/layout.tsx

```diff
@@ -1,53 +1,33 @@
 import "./globals.css";
 import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import { ThemeProvider } from "@/components/theme-provider";
+import { RootNavMenu } from "./root-nav-menu";
 
-import { Toaster } from "@/components/ui/toaster";
-import { cn } from "@/lib/utils";
-import { ThemeProvider } from "@/components/theme/provider";
-
-// TODO #15
-// The shadcn nav menu does not seem to be working
-// import {
-//   NavigationMenu,
-//   NavigationMenuContent,
-//   NavigationMenuIndicator,
-//   NavigationMenuItem,
-//   NavigationMenuLink,
-//   NavigationMenuList,
-//   NavigationMenuTrigger,
-//   NavigationMenuViewport,
-// } from "@/components/ui/navigation-menu";
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: "Cong Manager",
-  description: "",
-
-  themeColor: [
-    { media: "(prefers-color-scheme: light)", color: "white" },
-    { media: "(prefers-color-scheme: dark)", color: "black" },
-  ],
+  title: "Create Next App",
+  description: "Generated by create next app",
 };
 
-export interface RootLayoutProps {
+export default function RootLayout({
+  children,
+}: {
   children: React.ReactNode;
-}
-
-export default function RootLayout({ children }: RootLayoutProps) {
+}) {
   return (
     <html lang="en">
-      <head />
-      <body className={cn("min-h-screen bg-background font-sans p-4")}>
+      <body className={inter.className}>
         <ThemeProvider
           attribute="class"
           defaultTheme="dark"
           enableSystem
           disableTransitionOnChange
         >
-          <div className="relative flex min-h-screen flex-col">
-            <div className="flex-1">{children}</div>
-            {/* SITE FOOTER */}
-            <Toaster />
-          </div>
+          <RootNavMenu />
```

Comment:
Try to use this design pattern (if in a SSR component, any client-side rendered components need to be components in another file)

Reply:
ahh ok, will do